mne/utils/misc.py

```diff
@@ -149,7 +149,7 @@ def run_subprocess(command, return_code=False, verbose=None, *args, **kwargs):
                     break
                 else:
                     err = err.decode('utf-8')
-                    logger.warning(err)
+                    warn(err)
```

Comment:
This seems to confuse the tests. I don't really see how, because the traceback is not very informative: https://github.com/mne-tools/mne-python/pull/9434/checks?check_run_id=2704866135

Except for three failures where coverage.py warns about something, it always looks like this:

```
 <decorator-gen-2>:24: in run_subprocess
    ???
mne/utils/misc.py:152: in run_subprocess
    warn(err)
mne/utils/_logging.py:358: in warn
    warnings.warn_explicit(
E   RuntimeWarning:
```

Do you have an idea what's going on @larsoner?

Reply:
Warnings are treated as errors in tests. Now you are emitting a warning, rather than logging with a WARNING level of verbosity, hence it raises an error. The easiest thing is probably to add an ignore line to `mne/conftest.py` for this "include is ignored because"

Reply:
... actually in this case I think `logger.warn` is arguably the correct behavior since this is basically an advanced wrapper for `subprocess`. I think we should restore the `logger.warning` for this one to mirror the `logger.info` that is used for `stdout`-level messages.

Reply:
thanks, that makes sense. Also thanks for directly pushing those commits. It all passes now.